TELCODOCS-260: Adding back PTP Tech Preview notice

aireilly opened the pull request:
main, enterprise-4.9 update for PTP for the following doc epics:

https://issues.redhat.com/browse/TELCODOCS-260
https://issues.redhat.com/browse/TELCODOCS-262

PTP Tech Preview notice was removed in error. PTP is still Tech Preview for 4.9

jboxman commented:
@aireilly where'd you find this? I can't find it. I have this instead:

`networking/configuring-ptp.adoc`

And:

```
:FeatureName: Precision Time Protocol (PTP) hardware
include::modules/technology-preview.adoc[leveloffset=+1]
```

Moved with this commit: `3cb986c5b0b97a0e62bfee2cf470dae6e1f55d64`

aireilly commented:
I renamed configuring-ptp.adoc > using-ptp.adoc to reflect the wider scope of the assembly and removed the TP notice in error. PTP is staying TP in 4.9.

ctauchen commented:
/lgtm

vikram-redhat commented:
/cherrypick enterprise-4.9
